packages/spartan/extra/gen_resolvconf.py

```diff
@@ -51,6 +51,7 @@ def check_server(addr):
 
 options timeout:1
 options attempts:3
+options edns0
```

Comment:
We may list external servers in `/etc/resolv.conf` which we don't control, so this option can't be used.